utils/coreutils/techutils.go

```diff
@@ -78,23 +88,37 @@ func DetectTechnologies(path string, isCiSetup, recursive bool) (map[Technology]
 	}
 	detectedTechnologies := make(map[Technology]bool)
 	for _, file := range filesList {
-		techName := detectTechnologyByFile(strings.ToLower(file), isCiSetup)
-		if techName != "" {
+		techNames := detectTechnologiesByFile(strings.ToLower(file), isCiSetup)
+		for _, techName := range techNames {
 			detectedTechnologies[techName] = true
 		}
 	}
 	return detectedTechnologies, nil
 }
 
-func detectTechnologyByFile(file string, isCiSetup bool) Technology {
+func detectTechnologiesByFile(file string, isCiSetup bool) (detected []Technology) {
+	detected = []Technology{}
 	for techName, techData := range technologiesData {
 		if isCiSetup == false || (isCiSetup && techData.ciSetupSupport) {
 			for _, indicator := range techData.indicators {
 				if strings.Contains(file, indicator) {
-					return techName
+					detected = append(detected, techName)
 				}
 			}
 		}
 	}
-	return ""
+	return detected
+}
+
+// DetectTechnologiesToString returns a string that includes all the names of the detected technologies separated by a comma.
+func DetectedTechnologiesToString(detected map[Technology]bool) string {
```

Comment:
This function seem to be unused by jfrog-cli-core. Let's just make sure it is used by jfrog-cli, and if not, remove it.